appshell/cefclient_mac.mm

```diff
@@ -334,8 +334,15 @@ - (void)windowWillEnterFullScreen:(NSNotification *)notification {
     }
     if (customTitlebar) {
         [customTitlebar setHidden:YES];
+        NSWindow *window = [notification object];
+
+        // Since we have nuked the title, we will have
+        // to set the string back as we are hiding the
+        // custom title bar.
+        [window setTitle:[customTitlebar titleString]];
```

Comment:
And on OSX 10.11.4
![image](https://cloud.githubusercontent.com/assets/2939847/15225195/f203c0fe-189a-11e6-8dd3-46a312be42ac.png)